code-challenges.js

```diff
@@ -59,3 +132,15 @@ const number2 = 100
 // Set two:
 const number3 = 27
 const number4 = 24
+
+ const subtraction = (num1, num2) =>{
+     if(num1 >= num2){
+         return num1 - num2
+     }else{
+         return num2 - num1
+     }
+ }
+
+ console.log(subtraction(number1, number2))
+ console.log(subtraction(number3, number4))
```

Comment:
Excellent work in this section!  Code is well written and your pseudocode is outstanding!